docs/contributing/index.rst

```diff
@@ -1,13 +1,141 @@
+
+      If the new AST Node should be disabled by default, add a ``visit_<AST Node>`` method such as the following:
+
+      .. code-block:: python
+
+          def visit_<AST Node>(self, node):
+              """`<AST Node>` expression currently not allowed."""
+              self.not_allowed(node)
+
+      Please note, that for all AST Nodes without an explicit ``visit_<AST Node>`` method, a default applies which denies the usage of this expression and functionality.
+      As we try to be **as explicit as possible**, all language features should have a corresponding ``visit_<AST Node>``.
+      That follows the Zen of Python:
+
+      .. code-block:: pycon
+          :emphasize-lines: 5
+
+          >>> import this
+          The Zen of Python, by Tim Peters
+
+          Beautiful is better than ugly.
+          Explicit is better than implicit.
+          Simple is better than complex.
+          Complex is better than complicated.
+          Flat is better than nested.
+          Sparse is better than dense.
+          Readability counts.
+          Special cases aren't special enough to break the rules.
+          Although practicality beats purity.
+          Errors should never pass silently.
+          Unless explicitly silenced.
+          In the face of ambiguity, refuse the temptation to guess.
+          There should be one-- and preferably only one --obvious way to do it.
+          Although that way may not be obvious at first unless you're Dutch.
+          Now is better than never.
+          Although never is often better than *right* now.
+          If the implementation is hard to explain, it's a bad idea.
+          If the implementation is easy to explain, it may be a good idea.
+          Namespaces are one honking great idea -- let's do more of those!
+
+  * Add a corresponding changelog entry.
+  * Additionally modify ``.meta.toml`` and run the ``meta/config`` script (for details see: https://github.com/mgedmin/check-python-versions) to update the following files:
+
+    * ``/setup.py`` - Check that the new Python version classifier has been added ``"Programming Language :: Python :: <version>",``, and that the ``python_requires`` section has been updated correctly.
+    * ``/tox.ini`` - Check that a ``testenv`` entry is added to the general ``envlist`` statement.
+    * ``/.github/workflows/tests.yml`` - Check that a corresponding Python version entry has been added to the matrix definition.
+    * ``/docs/conf.py`` - Add the Python version to the ``intersphinx_mapping`` list.
+
+  * Run the tests, linting and docs generation via ``tox`` locally before push.
+
+* Create a pull request.
+
+Enable a Python Feature in RestrictedPython
++++++++++++++++++++++++++++++++++++++++++++
+
+To enable a certain functionality in RestrictedPython, do the following:
+
+#. `Create a new issue on GitHub <https://github.com/zopefoundation/RestrictedPython/issues/new/choose>`__, requesting the new feature, for discussion.
```

Comment:
The issue chooser needs to have an option to request a new feature. It's not obvious what to do because the two big friendly green buttons to report a bug and a security issue dominate the UI. There is also a link to add a new default issue template for the Zope organization, which can be overridden in each repository as needed. We do this for the Plone foundation.

![issue-chooser](https://user-images.githubusercontent.com/102112/196608985-5b225bca-d4c5-46e0-9c85-209daaa148b5.png)

Reply:
Yes, but that is not part of a pull request, we need a organisational work for that, @icemac can you add a new template, or should we add a feature request tempalte to all zopefoundation repos?

Reply:
Feel free to suggest a generic template that can be used across the organization if at all possible. It becomes a maintenance nightmare if there is a custom template in some or all repositories.

Reply:
Issue created: https://github.com/zopefoundation/RestrictedPython/issues/242